### PR TITLE
T/haproxylogging

### PR DESCRIPTION
**Pull request by gilleyj**
https://github.com/easybib/issues/issues/4152

### CODE REVIEW

**Comment by gilleyj** [comment]
@fh I'm not sure how I can test this with out wrecking stuff in OpsWorks.  However I have tested and it does setup and configure the resources in vagrant.

**Comment by gilleyj** [comment]
@fh it appears to work now.  I'm a bit confused as to when the haproxy::logs recipe is called but it does work when I turned up the lb1 on citation pg.

File: haproxy/recipes/configure.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 template '/etc/haproxy/haproxy.cfg' do
   source 'haproxy.easybib.cfg.erb'
+  # source 'haproxy.cfg.frontend.erb'
```
Comment (fh) [inline haproxy/recipes/configure.rb]: remove this :)

File: haproxy/recipes/logs.rb [hunk 2]
```diff
@@ -1,13 +1,21 @@
-if (node.fetch('lsb', {})['codename'] == 'precise')
+if node.fetch('lsb', {}).fetch('codename', {}) == 'precise' || node.fetch('lsb', {}).fetch('codename', {}) == 'trusty'
   # trusty has different log routing, it wont work there
```
Comment (fh) [inline haproxy/recipes/logs.rb]: Remove this

**Pull request merged**